odl/solvers/functional/functional.py

```diff
@@ -686,7 +693,12 @@ def __init__(self, func, vector):
                             ''.format(func))
 
         OperatorRightVectorMult.__init__(self, operator=func, vector=vector)
-        Functional.__init__(self, space=func.domain)
+        # HELP: before it was Functional.__init__(self, domain=func.domain).
+        # But doesn't this loose the information on grad_lipschitz and linear?
```

Comment:
Yes, it seems like the previous implementation forgets the linearity and lipschitz constant for the gradient. However, wouldn't the latter change for this functional? Anyway, we are thinking of removing `grad_lipschitz` since it is not currently used anywhere. See #1300

Reply:
You are right and if I am not mistaken, it would change in a nontrivial way. My guess for the upper bound would be something like `func.grad_lipschitz * ||constant||_\infty`? Should I implement that or will it be scrapped anyways?

Reply:
Wait... This only works for linear operators, correct? So just ignore it and make it `NaN`?